examples/server_certificate.tf

```diff
@@ -0,0 +1,43 @@
+provider "oneview" {
+	ov_password = "<password>"
+	ov_endpoint = "<endpoint>"
+	ov_sslverify = false
+	ov_apiversion = <ov-apiversion>
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
+        name = "test_scope"
+}
+
+resource "oneview_server_certificate" "ServerCertificate" {
+
```

Comment:
Format the file properly without many empty lines and spaces.

Reply:
Done